### DISCUSSION TITLE
fType metadata member name seems to be not allowed

### DISCUSSION THREAD

**Discussion by jgalan** [rest-for-physics/framework#146] [General]
When trying to use `fType` in one of my specific metadata classes, it seems it does not like it.

I was a bit surprised, but if I change the name it solves the problem.

It makes sense? @nkx

**Comment by nkx111** [comment]
Yes, the key words `name`, `title`, `verboselevel`, `type`, `value` and `store` are omitted in auto parameter loading.

https://github.com/rest-for-physics/framework/blob/master/source/framework/core/src/TRestMetadata.cxx#L2330
